packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -1588,87 +1600,28 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
     return await withLock(this.#vaultOperationMutex, callback);
   }
 
-  /**
-   * Serialize the encryption key and authentication key pair.
-   *
-   * @param encKey - The encryption key to serialize.
-   * @param pwEncKey - The password encryption key to serialize.
-   * @param authKeyPair - The authentication key pair to serialize.
-   * @returns The serialized encryption key and authentication key pair.
-   */
-  #serializeKeyData(
-    encKey: Uint8Array,
-    pwEncKey: Uint8Array,
-    authKeyPair: KeyPair,
-  ): {
-    toprfEncryptionKey: string;
-    toprfPwEncryptionKey: string;
-    toprfAuthKeyPair: string;
-  } {
-    const b64EncodedEncKey = bytesToBase64(encKey);
-    const b64EncodedPwEncKey = bytesToBase64(pwEncKey);
-    const b64EncodedAuthKeyPair = JSON.stringify({
-      sk: bigIntToHex(authKeyPair.sk), // Convert BigInt to hex string
-      pk: bytesToBase64(authKeyPair.pk),
-    });
-
-    return {
-      toprfEncryptionKey: b64EncodedEncKey,
-      toprfPwEncryptionKey: b64EncodedPwEncKey,
-      toprfAuthKeyPair: b64EncodedAuthKeyPair,
-    };
-  }
-
   /**
    * Parse and deserialize the authentication data from the vault.
    *
    * @param data - The decrypted vault data.
    * @returns The parsed authentication data.
    * @throws If the vault data is not valid.
    */
-  #parseVaultData(data: unknown): {
-    toprfEncryptionKey: Uint8Array;
-    toprfPwEncryptionKey: Uint8Array;
-    toprfAuthKeyPair: KeyPair;
-    revokeToken?: string;
-    accessToken: string;
-  } {
+  #parseVaultData(data: unknown): VaultData {
     if (typeof data !== 'string') {
-      throw new Error(
-        SeedlessOnboardingControllerErrorMessage.InvalidVaultData,
-      );
+      throw new Error(SeedlessOnboardingControllerErrorMessage.VaultDataError);
     }
 
     let parsedVaultData: unknown;
     try {
       parsedVaultData = JSON.parse(data);
     } catch {
-      throw new Error(
-        SeedlessOnboardingControllerErrorMessage.InvalidVaultData,
-      );
+      throw new Error(SeedlessOnboardingControllerErrorMessage.VaultDataError);
```

Comment:
Changing the error type can be considered a breaking change.
Why did you change it?